packages/neos-ui-ckeditor5-bindings/src/manifest.config.js

```diff
@@ -147,5 +144,21 @@ export default ckEditorRegistry => {
             ]}
     }));
 
+    //
+    // @see https://ckeditor.com/docs/ckeditor5/16.0.0/api/module_core_editor_editorconfig-EditorConfig.html#member-placeholder
+    //
+    config.set('placeholder', (config, {globalRegistry, editorOptions}) => {
```

Comment:
Would be helpful to have a note here how we integrate the native placeholder. For me it was directly visible.

Reply:
Adjusted ;)